Domain-Setup.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Obtain A Domain Name and Redirect Domain DNS
+layout: topic
+categories: guide
+---
+
+Now that you have a webpage and are ready to deploy it, you need to 
+set up a domain name for your website. This wiki will walk you through 
+that process.
+
+## Step 1: Obtain a Domain Name
+
+It is recommended to get a domain name off of [NameCheap](https://www.namecheap.com), 
+but you can purchase a domain name from other providers (including 
+[GoDaddy](https://www.godaddy.com/) and [HostGator](https://www.hostgator.com/domains)).
+
+To get your domain name:
+- Create an account with the provider of your choosing.
+- Pick a name that sounds good to you. NameCheap has convenient search 
+  functionality to help you find the perfect domain name for your 
+  project.
+- **For NameCheap:** select [WhoIsGuard](https://www.namecheap.com/support/knowledgebase/article.aspx/278/37/what-is-whoisguard) 
+as an option when you are going through checkout. This is a free 
+service that NameCheap provides that protects your personal information 
+when other people run `whois` on your domain name.
+- Purchase the domain name.
+
+And you're done!
+
+_Note: avoid purchasing extra paid services (like NameCheap's PremiumDNS 
+service); these add-on purchases are not necessary with adequate site 
+configuration and a little know-how._
+
+## Step 2: Forward the Domain Name to a DNS Provider
+
+If you are using Digital Ocean to host your site (as recommended in 
+the [hackathon docs](Server-Creation)), you can set up domain name 
+system (DNS) forwarding to Digital Ocean directly. The required steps 
+are described in detail [here](https://www.digitalocean.com/community/tutorials/how-to-point-to-digitalocean-nameservers-from-common-domain-registrars).
+
+Here's a quick run-down of the steps involved:
+- Log into your domain purchasing site (in this example, we'll use 
+  NameCheap).
+- Go to the management page for the domain you bought. In NameCheap, 
+  this will be under [Dashboard](https://ap.www.namecheap.com/dashboard).
+- Under the domain you bought, click "Manage".
+- Under the "Nameservers" section, select "Custom DNS" in the dropdown, 
```

Comment:
I believe that namecheap also makes you manually turn off the parking page, something that could be a "gotcha" for newbies.

Reply:
What do you mean by parking page? Am confuse. Will look into this later

Reply:
The parking page is just the default page they show when you haven't set up the website. http://parkingpage.namecheap.com/

Reply:
I'm adding a note about it. TIL there's a switch at the bottom of the domain dashboard to turn off parking pages :sweat_smile:

Reply:
FWIW I haven't had issues with this to date

Reply:
It's bugged me once before. I wish that switch wasn't so out of the way.

Reply:
Addressed in d37e4b7fa52a0f04e55dffc0e103bda1ac99adf6